query/sources/common/src/main/java/org/codice/ddf/admin/sources/utils/SourceUtilCommons.java

```diff
@@ -178,19 +174,6 @@ public Document createDocument(String body)
         return sourceInfoListField;
     }
 
-    public Report saveSource(PidField pid, Map<String, Object> serviceProps, String factoryPid) {
```

Comment:
I think this class will need some more cleanup now that this has been removed. Doesn't look like we will need `ServiceCommons` in here anymore.